code/game/objects/structures/customplaque.dm

```diff
@@ -0,0 +1,123 @@
+/obj/structure/customplaque //This is a plaque you can craft with gold, then permanently engrave a title and description on, with a fountain pen.
+	icon = 'icons/obj/decals.dmi'
+	icon_state = "blankplaque"
+	name = "blank plaque"
+	desc = "A blank plaque. Use a fancy pen to engrave it, use a wrench to detatch it from the wall."
+	anchored = TRUE
+	opacity = 0
+	density = FALSE
+	layer = SIGN_LAYER
+	max_integrity = 200 //Twice as durable as regular signs.
+	armor = list("melee" = 50, "bullet" = 0, "laser" = 0, "energy" = 0, "bomb" = 0, "bio" = 0, "rad" = 0, "fire" = 50, "acid" = 50)
+	rad_flags = RAD_PROTECT_CONTENTS | RAD_NO_CONTAMINATE
+	var/unengraved = TRUE //Custom plaque structures and items both start "unengraved", once engraved with a fountain pen their text can't be altered again.
+
+/obj/item/customplaque
+	icon = 'icons/obj/decals.dmi'
+	icon_state = "blankplaque"
+	name = "blank plaque"
+	desc = "A blank plaque. Place it on a wall and use a fancy pen to engrave it, use a wrench to detatch it from the wall."
+	w_class = WEIGHT_CLASS_NORMAL
+	custom_materials = list(/datum/material/gold = 2000)
+	var/plaque_path = /obj/structure/customplaque //This points the item to make the proper structure when placed on a wall.
+	var/unengraved = TRUE
+
+/obj/structure/customplaque/play_attack_sound(damage_amount, damage_type = BRUTE, damage_flag = 0)
+	switch(damage_type)
+		if(BRUTE)
+			if(damage_amount)
+				playsound(src.loc, 'sound/weapons/smash.ogg', 80, TRUE)
```

Comment:
```suggestion
				playsound(loc, 'sound/weapons/smash.ogg', 80, TRUE)
```
`src` refers to the thing calling the proc. All variables without a `.` access and not in the call of the proc are implied to have `src.`, which makes it not necessary to add it.

Only add `.src` when the object variable has the same name as the proc call variable.
For example, if `/obj/structure/customplaque` has an object variable called `damage_amount` while `play_attack_sound` also has a variable of the same name.
In such case, in order to differentiate them you'd do `src.damage_amount` for the object variable and just `damage_amount` for the proc variable.

Reply:
Fixed.